syntax = "proto3";

package hello;

service Greeter {
    rpc SayHello (HelloReq) returns (HelloResp) {}
}

message HelloReq {
    string message = 1;
}

message HelloResp {
    string message = 1;
}